tests/unit/test_parser.py

```diff
@@ -236,6 +236,23 @@ def testNamedEmptyBlocks(self):
                                ['block', ('a', ' ', 'A'), [], 'a A'],
                                ['block', ('b', ' ', 'B /'), [], 'b B /']])
 
+    def testMultilineBlocks(self):
+        text = """\
+<long \
```

Comment:
Hmmm, these escapes inside `"""` - won't they be eaten by Python so that a single line ends up being in `text` variable? May be we should have double `\\`?